frontend/js/src/common/listens/MBIDMappingModal.tsx

```diff
@@ -285,21 +267,74 @@ export default NiceModal.create(({ listenToMap }: MBIDMappingModalProps) => {
                 </div>
               )}
             </div>
-            <div className="modal-footer">
-              <button
-                type="button"
-                className="btn btn-default"
-                onClick={closeModal}
-              >
-                Cancel
-              </button>
-              <button
-                type="submit"
-                className="btn btn-success"
-                disabled={!selectedRecording}
+            <div className="modal-footer" style={{ textAlign: "left" }}>
+              <div
+                style={{
+                  display: "flex",
+                  justifyContent: "space-between",
+                  alignItems: "center",
+                }}
```

Comment:
We can use the utility class `mb-10` here to add the same 10px bottom margin you had on the mass link button.
Also don't think we need `alignItems: "center"` here.
```suggestion
              <div
                className="mb-10"
                style={{
                  display: "flex",
                  justifyContent: "space-between",
                }}
```